microsoft/testsuites/network/sriov.py

```diff
@@ -542,173 +527,106 @@ def sriov_ethtool_offload_setting(self, environment: Environment) -> None:
-        source_ssh = source_node.tools[Ssh]
-        dest_ssh = dest_node.tools[Ssh]
-
-        dest_ssh.enable_public_key(source_ssh.generate_key_pairs())
-        # generate 200Mb file
-        source_node.execute("dd if=/dev/urandom of=large_file bs=100 count=0 seek=2M")
-        max_retry_times = 10
-        for _, source_nic_info in vm_nics[source_node.name].items():
-            matched_dest_nic_name = ""
-            for dest_nic_name, dest_nic_info in vm_nics[dest_node.name].items():
-                # only when IPs are in the same subnet, IP1 of machine A can connect to
-                # IP2 of machine B
-                # e.g. eth2 IP is 10.0.2.3 on machine A, eth2 IP is 10.0.3.4 on machine
-                # B, use nic name doesn't work in this situation
+            initial_pci_interrupts_by_cpus = (
+                client_interrupt_inspector.sum_cpu_counter_by_index(
+                    client_nic_info.pci_slot
+                )
+            )
+            matched_server_nic_info: NicInfo
+            for _, server_nic_info in vm_nics[server_node.name].items():
                 if (
-                    dest_nic_info.ip_addr.rsplit(".", maxsplit=1)[0]
-                    == source_nic_info.ip_addr.rsplit(".", maxsplit=1)[0]
+                    server_nic_info.ip_addr.rsplit(".", maxsplit=1)[0]
+                    == client_nic_info.ip_addr.rsplit(".", maxsplit=1)[0]
                 ):
-                    matched_dest_nic_name = dest_nic_name
+                    matched_server_nic_info = server_nic_info
                     break
-            assert_that(matched_dest_nic_name).described_as(
-                f"can't find the same subnet nic with {source_nic_info.ip_addr} on"
-                f" machine {source_node.name}, please check network setting of "
-                f"machine {dest_node.name}."
-            ).is_not_empty()
-            desc_nic_info = vm_nics[dest_node.name][matched_dest_nic_name]
-            dest_ip = vm_nics[dest_node.name][matched_dest_nic_name].ip_addr
-            source_ip = source_nic_info.ip_addr
-            source_synthetic_nic = source_nic_info.upper
-            dest_synthetic_nic = desc_nic_info.upper
-            source_nic = source_vf_nic = source_nic_info.lower
-            dest_nic = dest_vf_nic = desc_nic_info.lower
-
-            if remove_module or turn_off_vf:
-                source_nic = source_synthetic_nic
-                dest_nic = dest_synthetic_nic
-            if turn_off_vf:
-                source_node.execute(f"ip link set dev {source_vf_nic} down", sudo=True)
-                dest_node.execute(f"ip link set dev {dest_vf_nic} down", sudo=True)
-
-            # get origin tx_packets and rx_packets before copy file
-            source_tx_packets_origin = self._get_packets(source_node, source_nic)
-            dest_tx_packets_origin = self._get_packets(
-                dest_node, dest_nic, "rx_packets"
+            assert matched_server_nic_info
```

Comment:
Add message, if it's failed, what the runner should know.

Reply:
added

Reply:
Not seen it in latest push.

Reply:
updated